easybuild/easyblocks/m/motioncor2.py

```diff
@@ -142,3 +149,10 @@ def sanity_check_step(self):
         }
 
         super(EB_MotionCor2, self).sanity_check_step(custom_paths)
+
+    def sanity_check_rpath(self, *args, **kwargs):
+        """Custom implementation of RPATH sanity check: allow skipping via skip_rpath_sanity_check."""
```

Comment:
Please add comment on why we need to skip the rpath check

Reply:
Actually looking at this again I think this is a workaround for a broader issue with RPATH for binary installs, and it is not the right approach:
- we use RPATH on our supercomputer at SURF, and we usually fix the issues with binary install by customizing the easyconfig (or adding hooks) to run `patchelf` commands to add the missing rpaths in the binary.
- this is an issue for all binary installs. Ideally, the rpath check in the case of binary install would also automatically fix the missing rpaths using `patchelf`. However `patchelf` cannot be used in the easyblock as it is not necessarily available on the system. So we have to do it at the easyconfig level.
- In the case of the current PR, this custom `sanity_check_rpath` is in my opinion not the right solution. In fact this is a binary install, and the `EB_MotionCor2` class should derive from the `PackedBinary` class instead of directly from the `EasyBlock` class. Since `PackedBinary` skips the rpath check, that would remove the need for a custom `sanity_check_rpath`.

A real fix for binary installs using RPATH should in my opinion be implemented in the `Binary` class rather than in software specific easyblocks, so we should not change the `sanity_check_rpath` at this level.

@akesandgren: my suggestion is to make `EB_MotionCor2` derive from `PackedBinary` instead of `EasyBlock`, and remove the custom `sanity_check_rpath` completely. Does that sound good to you?

Reply:
Not sure, I need to check what PackedBinary does. Probably won't have time the coming days/week

Reply:
Lets consider that in a separate PR.